README.md

```diff
@@ -230,6 +213,35 @@ connection.close();
 connector.close();
 ```
 
+### Specifying Public or Private IP
```

Comment:
We can probably fold the `Using a Private Service Connect (PSC) IP` heading into this one to make things more concise. Maybe renaming it to `Specifying IP Address Type` like python connector PR?: https://github.com/GoogleCloudPlatform/cloud-sql-python-connector/pull/766/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5